summingbird-scalding/src/main/scala/com/twitter/summingbird/scalding/batch/BatchedStore.scala

```diff
@@ -93,14 +105,14 @@ trait BatchedStore[K, V] extends scalding.Store[K, V] { self =>
 
   protected def sumByBatches[K1, V: Semigroup](ins: TypedPipe[(Timestamp, (K1, V))],
     capturedBatcher: Batcher,
-    commutativity: Commutativity): TypedPipe[((K1, BatchID), (Timestamp, V))] = {
+    commutativity: Commutativity): TypedPipe[(LTuple2[K1, BatchID], (Timestamp, V))] = {
```

Comment:
this is weird. You have a protected method returning a private[batch] type. So, doesn't that effectively make this method `private[batch]` which may not be what we want, but maybe?